### PR TITLE
Add trades model of avalanche_c of Uniswap

### PR DESCRIPTION
**Pull request by lequangphu**
Copy from models/uniswap/base, replace base with avalanche_c, rename pair contract, add the new model to uniswap_trades and dbt_project

# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by Hosuke** [comment]
Hi @lequangphu, thank you for your contribution!
We can add some seed records for `uniswap_v3_avalanche_c_trades.trades` in this file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

File: models/uniswap/avalanche_c/uniswap_avalanche_c_trades.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{{ config(
+        alias = 'trades'
```
Comment (Hosuke) [inline models/uniswap/avalanche_c/uniswap_avalanche_c_trades.sql:2]: We may also need the schema here:
```suggestion
        schema = 'uniswap_avalanche_c',
        alias = 'trades'
```

File: models/uniswap/avalanche_c/uniswap_avalanche_c_schema.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+      - check_dex_seed:
+          blockchain: avalanche_c
+          project: uniswap
+          version: 3
```
Comment (Hosuke) [inline models/uniswap/avalanche_c/uniswap_avalanche_c_schema.yml:26]: Hi @lequangphu, in order for this `check_dex_seed` test to work,
please add some seed records in this file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/uniswap/avalanche_c/uniswap_avalanche_c_schema.yml (hunk 2 above)
Comment (lequangphu) [reply models/uniswap/avalanche_c/uniswap_avalanche_c_schema.yml:26]: sure, let me add, thanks!

Comment (lequangphu) [reply models/uniswap/avalanche_c/uniswap_avalanche_c_schema.yml:26]: the records should be added in the same PR with the model, right?

**Comment by lequangphu** [comment]
@Hosuke another error after adding trade records, pls help!
Screenshot 2023-11-11 at 11 03 25

File: seeds/dex/trades/dex_trades_seed.csv
Comment (Hosuke) [inline seeds/dex/trades/dex_trades_seed.csv:1]: By the way why does this file have so many lines changed?

File: seeds/dex/trades/dex_trades_seed.csv [hunk 3]
```diff
@@ -1,474 +1,479 @@
...
+avalanche_c,uniswap,3,2023-11-11,0x3fd0df30055592268a66399a4d958ae6aaa957cab7442c0ecc60d7257a156261,7,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,497.409883,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,34.17977416
+avalanche_c,uniswap,3,2023-11-11,0x31e1a422f13f742ba7ad4c5b2b33bf2cc6c05777cfff4062f8e9667dc0eab524,14,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,95.79678643,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,1379.142478
+avalanche_c,uniswap,3,2023-11-11,0x79cc2e2b8543ae7b5215f9f389069106061efc619fa23437b5662d81fac2d8dc,2,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,661.400762,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,46
+avalanche_c,uniswap,3,2023-11-11,0x31e1a422f13f742ba7ad4c5b2b33bf2cc6c05777cfff4062f8e9667dc0eab524,16,0x49d5c2bdffac6ce2bfdb6640f4f80f226bc10bab,0.6729956329,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,95.79678643
+avalanche_c,uniswap,3,2023-11-11,0x41b6ec0e3ba96f6c72bab7954353152ba44904bd61232f91e824c9a74495b1e4,2,0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7,600.415091,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,41.78982883
```
Comment (Hosuke) [inline seeds/dex/trades/dex_trades_seed.csv:479]: I will recover this file and add these 4 lines then, to keep this PR clean.

File: models/uniswap/avalanche_c/uniswap_avalanche_c_schema.yml (hunk 2 above)
Comment (Hosuke) [reply models/uniswap/avalanche_c/uniswap_avalanche_c_schema.yml:26]: Yes.

File: seeds/dex/trades/dex_trades_seed.csv (hunk 3 above)
Comment (lequangphu) [reply seeds/dex/trades/dex_trades_seed.csv]: what's wrong with my seed file? it will be helpful for my next PR.

Comment (Hosuke) [reply seeds/dex/trades/dex_trades_seed.csv]: You can see your modification lost precision in numbers for other seed records.
That's why I recovered modification on other seed records, and just add relevant lines.

File: models/_sector/dex/trades/avalanche_c/_schema.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      contributors: jeff-dude, masquot, soispoke, mtitus6
+    config:
+      tags: [ 'avalanche_c', 'dex', 'trades', 'uniswap', 'v3' ]
+    description: "uniswap avalanche_c v3 base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+
```
Comment (Hosuke) [inline models/_sector/dex/trades/avalanche_c/_schema.yml:18]:
```suggestion
    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - tx_hash
            - evt_index
      - check_dex_seed:
          blockchain: avalanche_c
          project: uniswap
          version: 3
          
```

File: seeds/dex/trades/dex_trades_seed.csv (hunk 3 above)
Comment (lequangphu) [reply seeds/dex/trades/dex_trades_seed.csv]: I edited the csv in google sheet. That might be the reason, thanks!

**Review by jeff-dude — APPROVED** [review]
thank you both!

**Pull request merged**